# __init__.py 2023/10/22 20:13
from .IllegalArgumentException import IllegalArgumentException

